### DISCUSSION TITLE
Renewal of community licence not possible, because compression enabled in one Database

### DISCUSSION THREAD

**Discussion by w0pr** [ravendb/ravendb#15558] [General]
Cannot access Studio because of expired license (License ID: 5c7848d6-28f6-458b-a2c8-a9cde2d35d36)

This is Server version 5.4.4.

I had the same problem 2 years ago, back then it seemed to have fixed itself somehow?
https://groups.google.com/g/ravendb/c/9j03ZQtS864/m/r8kIwqd_BgAJ

I was sure I had disabled compression, but it somehow came back...

![image](https://user-images.githubusercontent.com/8257055/210612043-86eb2783-5844-445c-8398-8934077e5acf.png)

UPDATE: After (Windows) Service Restart, seems to have automatically renewed cluster service certificate and community license (ID as above) extended by one year.

Studio access restored (because license extension).

Curiously, there was no database that had a Collection in DocumentCompression (checked DatabaseRecord of all databases), only "CompressRevisions" was enabled on one.

![image](https://user-images.githubusercontent.com/8257055/210639948-4f973b50-bd41-486a-8e30-d826edc26fe1.png)

**Comment by ayende** [comment]
Yes `CompressRevision` is this feature, which should _not_ be enabled in community licenses.
I would recommend disabling that to avoid having this again next year.

**Reply by w0pr** [reply]
Last time, I disabled DocumentCompression for the one Collection I had it enabled for (when it was still possible to do so on Community license), but I must have missed "CompressRevisions"! 

Done now and it is not possible to reenable any DocumentCompression toggle with Community license. Thanks.
